jobs/riemann/templates/config/memory.clj.erb

```diff
@@ -0,0 +1,31 @@
+(info "Loading memory alerts")
+
+(defn memory-alerts [pd]
+  (info "Setting up memory alerts")
+  (where (service "memory/percent-used")
+    (smap
+      (fn [event]
+        ; set the event's state based on the value of the metric, and possibly the specific host it originated from
+        (assoc event :state
+          (cond
+          <% p("riemann.memory.overrides").each do |over| %>
+            (and (re-matches #"<%= over["host"] %>" (:host event)) (> (:metric event) <%= over["threshold"] %>)) "<%= over["state"] %>"
```

Comment:
If we're using `re-matches`, we might want to mention in the docs that the regex has to match the entire host. Or we can use `re-find`.